be/src/service/internal_service.cpp

```diff
@@ -983,7 +983,14 @@ void PInternalServiceImpl::transmit_block(google::protobuf::RpcController* contr
                                           google::protobuf::Closure* done) {
     int64_t receive_time = GetCurrentTimeNanos();
     response->set_receive_time(receive_time);
-    PriorityThreadPool& pool = request->has_block() ? _heavy_work_pool : _light_work_pool;
+
+    if (!request->has_block() && config::brpc_light_work_pool_threads == -1) {
```

Comment:
in which case？the config::brpc_light_work_pool_threads  == -1？

Reply:
> in which case？the config::brpc_light_work_pool_threads == -1？

default it -1